addons/danger/functions/fnc_tacticsContact.sqf

```diff
@@ -26,6 +26,12 @@ if (isNull _enemy) then {
     _enemy = _unit findNearestEnemy _unit;
 };
 
+// no enemy -- minor pause
+if ((side _unit) isEqualTo (side _enemy)) exitWith {
+    (group _unit) setVariable [QGVAR(contact), time + 10 + random 10];
```

Comment:
Small delay to prevent constant checking for contact.  Not too long however, so if an enemy appears. The unit will still appropriately react.